### ISSUE TITLE
The Rope doesn't tear when an explosion happens alongside

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#39]
```
What steps will reproduce the problem?
1. Shoot a rope
2. Arm a mine
3. Dont get caught in the explosion, but have the rope in the way

What is the expected output? What do you see instead?
The Rope to tear/cut loose

What version of the product are you using? On what operating system?
0.9.13, Win7

Please provide any additional information below.
http://www.youtube.com/watch?v=wKlFe3QhAAA

```

-----
Original issue reported on code.google.com by `joship...@gmail.com` on 15 Sep 2010 at 10:23

**Comment by GoogleCodeExporter** [comment]
```
As with the other rope issues you posted, this is one of the issues we didn't 
touch so far as at least some consider it being a part of the game (i.e. 
intended). Personally I'm not sure about it and tearing the rope might be an 
option but that's nothing I'd decide right away without more opinions.

Setting the status to "Accepted" for now but this doesn't mean we'll take 
action immediately (if at all).
```

-----
Original comment by `mario.li...@gmail.com` on 16 Sep 2010 at 9:11
* Changed state: **Accepted**
* Added labels: **Priority-Low**
* Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4386:ed78465973f6

```

-----
Original comment by `kyberneticist@gmail.com` on 21 Nov 2010 at 6:21
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I think this issue got confused with issue 40, but I don't think we plan on 
making the rope tear on damage.
```

-----
Original comment by `sheepyluva` on 28 Mar 2011 at 7:33
* Changed state: **WontFix**
